src/redux/store.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+import authorsSlice from './authors/authorsSlice';
+import booksSlice from './books/booksSlice';
+
+const store = configureStore({
+  reducer: {
+    books: booksSlice,
+    authors: authorsSlice,
```

Comment:
After implementing the suggestions above, you should remember to work on this too and the imports also😉.